database/data/seed_data.js

```diff
@@ -0,0 +1,120 @@
+const lorem = require('lorem-ipsum');
+
+const options = require('../lorem_options');
+const dogBreeds = require('../dog_breeds.js');
+
+
+// ex. randomInt(1,5) => generates 1-5 inclusive
+let randomInt = (min, max) => {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+let breedTypes = ['Herding', 'Sporting', 'Non-Sporting', 'Working', 'Hounds', 'Terriers', 'Toy'];
+
+let breedInfoArray = []
+
+//will generate random numbers, categories or ipsum data depending on breed attribute
+for (let i = 1; i <=100; i++) {
+
+  let randomWeightMin = randomInt(5, 75);
+  let randomWeightMax = randomWeightMin + randomInt(5, 50);
```

Comment:
if you wanted to make this more realistic for a range, could generate the max weight as a percentage of the min weight instead of an absolute  number. For example, say the max weight is somewhere between 15 - 35% more than the min weight (or look at petfinder to get an idea of what this percentage usually is)